lib/Driver/Compilation.cpp

```diff
@@ -679,6 +680,11 @@ namespace driver {
     }
 
     /// Schedule all jobs we can from the initial list provided by Compilation.
+    ///
+    /// It would probably be safe and simpler to markTransitive on the start
+    /// nodes in the "Always" condition from the start instead of using
+    /// markIntransitive and having later functions call markTransitive. That
+    /// way markInstransitive would be an implemenation detail.
```

Comment:
This does not belong in a doc comment because it's not relevant to callers. Maybe it belongs next to markIntransitive instead?

(Also typo: "implemenation".)

Reply:
Great! Done.